controls/badgeview/overview.md

```diff
@@ -23,6 +23,8 @@ Badges can be used as an additional marker for any element: to decorate avatars,
 * [Alignment]({%slug badgeview-position-alignment%}#badge-alignment)&mdash;You can align the Badge indicator according to its content.
 * [Animation]({%slug badgeview-animation%})&mdash;You can choose whether to have an `Animation` while displaying the Badge indicator. In addition, you can specify the animation duration and animation easing.
 * [Badge types]({%slug badgeview-predefined-badges%})&mdash;You can select the type of a Badge from a set of various predefined options. Also, you can customize and style each Badge type.
+* Change the [visibility of the badge indicator]({%slug badgeview-configuration%}#badge-visibility) to collapsed, hidden and visible. 
```

Comment:
When editing document sections, take a look at the already available content. Then add the new content by using the same style. In this case, the previously added items use the following pattern:
`* [Short feature name]&mdash;Short feature description`.